tfx/tools/cli/handler/beam_handler_test.py

```diff
@@ -357,7 +357,7 @@ def testCreateRun(self):
     handler = beam_handler.BeamHandler(flags_dict)
     with self.captureWritesToStream(sys.stdout) as captured:
       handler.create_run()
-    self.assertIn("[%s, '%s']" % (sys.executable, self.pipeline_path),
+    self.assertIn("['%s', '%s']" % (sys.executable, self.pipeline_path),
```

Comment:
There is an issue with this if sys.executable is None or something non-string.
```suggestion
    self.assertIn("%s" % [sys.executable, self.pipeline_path],
```